scripts/deployment/goerli-testnets.json

```diff
@@ -0,0 +1,61 @@
+{
+  "L1": {
+    "name": "Goerli",
+    "rpc": "https://goerli.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161",
+    "chain_id": 5
+  },
+  "L2": [
+    {
+      "name": "Arbitrum Goerli",
+      "rpc": "https://goerli-rollup.arbitrum.io/rpc",
+      "chain_id": 421613,
+      "l1_messenger": [
+        "ArbitrumL1Messenger",
+        "0xaf4159A80B6Cc41ED517DB1c453d1Ef5C2e4dB72",
+        "0x6BEbC4925716945D46F0Ec336D5C2564F419682C"
+      ],
+      "l2_messenger": "ArbitrumL2Messenger",
+      "finality_period": 604800,
+      "request_manager_arguments": {
+        "claim_stake": 0.00047,
+        "claim_request_extension": 3600,
+        "claim_period": 3600,
+        "challenge_period_extension": 3600
```

Comment:
Suggestion: 
   "claim_stake": 0.00047,
    "claim_request_extension": 900,
    "claim_period": 900,
    "challenge_period_extension": 900

In order to better test challenges I suggest to lower the values to 900 (15 minutes)

Reply:
✅